config/secrets.py.example

```diff
@@ -24,6 +24,19 @@ elif LOCALHOST:
     UPLOADS_DEFAULT_DEST = BASE_DIR + '/uploads/'
     UPLOADS_DEFAULT_URL = 'http://localhost:5000/uploads/'
 
+
+# Database backup path. PRODUCTION and STAGING can be ignored for
+# local testing.
+if PRODUCTION:
+    BACKUP_DEFAULT_DEST = ''
+    BACKUP_DEFAULT_URL = ''
```

Comment:
`BACKUP_DEFAULT_URL` isn't used, right? 
(And I imagine we don't want to expose this to the web)